### PR TITLE
[ADD] pos_order_count_store

### PR DESCRIPTION
**Pull request by robinkeunen**
- stores pos_order_count field to use it in filters and pivot view
- adds pos_order_ids field to use it in pivot view

Already discussed in https://github.com/beescoop/Obeesdoo/pull/468

### CODE REVIEW

**Comment by victor-champonnois** [comment]
@legalsylvain for a follow up from https://github.com/beescoop/Obeesdoo/pull/468

**Review by victor-champonnois — APPROVED** [review]
LGTM

**Review by legalsylvain — APPROVED** [review]

**Review by ivantodorovich — APPROVED** [review]

File: pos_order_count_store/README.rst [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/beescoop/Obeesdoo/issues>`_.
```
Comment (ferdymercury) [inline pos_order_count_store/README.rst:32]: Maybe the links in this section need to be changed to oca?

File: pos_order_count_store/README.rst [hunk 2]
```diff
@@ -0,0 +1,58 @@
+=====================
+Beesdoo POS Reporting
```
Comment (ferdymercury) [inline pos_order_count_store/README.rst:2]: Beesdoo can be omitted?

File: pos_order_count_store/README.rst [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+Contributors
+~~~~~~~~~~~~
+
+* BEES coop - Cellule IT
+* Coop IT Easy SC
+
+Maintainers
+~~~~~~~~~~~
+
+This module is part of the `beescoop/Obeesdoo <https://github.com/beescoop/Obeesdoo/tree/12.0/beesdoo_pos_reporting>`_ project on GitHub.
```
Comment (ferdymercury) [inline pos_order_count_store/README.rst:56]: link update?

File: pos_order_count_store/README.rst [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-beescoop%2FObeesdoo-lightgray.png?logo=github
+    :target: https://github.com/beescoop/Obeesdoo/tree/12.0/beesdoo_pos_reporting
```
Comment (ferdymercury) [inline pos_order_count_store/README.rst:17]: link?

File: pos_order_count_store/static/description/index.html [hunk 5]
```diff
@@ -0,0 +1,415 @@
...
+<div class="section" id="contributors">
+<h2><a class="toc-backref" href="#id4">Contributors</a></h2>
+<ul class="simple">
+<li>Beescoop - Cellule IT</li>
+<li>Coop IT Easy SC</li>
+</ul>
+</div>
+<div class="section" id="maintainers">
+<h2><a class="toc-backref" href="#id5">Maintainers</a></h2>
+<p>This module is part of the <a class="reference external" href="https://github.com/beescoop/obeesdoo/tree/12.0/beesdoo_pos_reporting">beescoop/obeesdoo</a> project on GitHub.</p>
```
Comment (ferdymercury) [inline pos_order_count_store/static/description/index.html:409]: link?

Comment (ferdymercury) [reply pos_order_count_store/static/description/index.html:409]: (and several more in this document)

Comment (robinkeunen) [reply pos_order_count_store/static/description/index.html:409]: Hi @ferdymercury Thanks for the review. All the "beesdoo leftover" you raised will be erased by the OCABOT when it generates new readme.rst and index.html on merge.

**Pull request merged**